gecko-revision

```diff
@@ -1 +1 @@
-7acfe5fcdb611518ecc36f5720cb784fdda1ad08
+wasi-ff140
```

Comment:
just to note that this is a branch name and not a commit - which means that the build could end up using a different version of bytecodealliance/firefox if someone pushes to this branch

Reply:
yes, that'd be really nice :D

Reply:
I changed things to using that exact tag name now. Thank you for raising this! <3